docs/standard/security/encrypting-data.md

```diff
@@ -1,262 +1,157 @@
+
+The managed symmetric cryptography classes are used with a special stream class called a <xref:System.Security.Cryptography.CryptoStream> that encrypts data read into the stream. The **CryptoStream** class is initialized with a managed stream class, a class that implements the <xref:System.Security.Cryptography.ICryptoTransform> interface (created from a class that implements a cryptographic algorithm), and a <xref:System.Security.Cryptography.CryptoStreamMode> enumeration that describes the type of access permitted to the **CryptoStream**. The **CryptoStream** class can be initialized using any class that derives from the <xref:System.IO.Stream> class, including <xref:System.IO.FileStream>, <xref:System.IO.MemoryStream>, and <xref:System.Net.Sockets.NetworkStream>. Using these classes, you can perform symmetric encryption on a variety of stream objects.  
   
- The following example illustrates how to create a new instance of the <xref:System.Security.Cryptography.RijndaelManaged> class, which implements the Rijndael encryption algorithm, and use it to perform encryption on a **CryptoStream** class. In this example, the **CryptoStream** is initialized with a stream object called `myStream` that can be any type of managed stream. The **CreateEncryptor** method from the **RijndaelManaged** class is passed the key and IV that are used for encryption. In this case, the default key and IV generated from `rmCrypto` are used. Finally, the **CryptoStreamMode.Write** is passed, specifying write access to the stream.  
+The following example illustrates how to create a new instance of the default implementation class for the <xref:System.Security.Cryptography.Aes> algorithm. The instance is used to perform encryption on a **CryptoStream** class. In this example, the **CryptoStream** is initialized with a stream object called `myStream` that can be any type of managed stream. The **CreateEncryptor** method from the **Aes** class is passed the key and IV that are used for encryption. In this case, the default key and IV generated from `aes` are used.
   
 ```vb  
-Dim rmCrypto As New RijndaelManaged()  
-Dim cryptStream As New CryptoStream(myStream, rmCrypto.CreateEncryptor(rmCrypto.Key, rmCrypto.IV), CryptoStreamMode.Write)  
+Dim aes As Aes = Aes.Create()  
+Dim cryptStream As New CryptoStream(myStream, aes.CreateEncryptor(aes.Key, aes.IV), CryptoStreamMode.Write)  
 ```  
   
 ```csharp  
-RijndaelManaged rmCrypto = new RijndaelManaged();  
-CryptoStream cryptStream = new CryptoStream(myStream, rmCrypto.CreateEncryptor(), CryptoStreamMode.Write);  
+Aes aes = Aes.Create();  
+CryptoStream cryptStream = new CryptoStream(myStream, aes.CreateEncryptor(), CryptoStreamMode.Write);  
```

Comment:
```suggestion
CryptoStream cryptStream = new CryptoStream(myStream, aes.CreateEncryptor(key, iv), CryptoStreamMode.Write);  
```

Making the C# match the VB.  (There is a zero-argument CreateEncryptor/CreateDecryptor, which is, effectively, "use the key and IV you already know about"; but the comment seems like it should match the sample... and the C# and VB should do the same mechanical steps)